### DISCUSSION TITLE
Config Help: append text to default onboarding PR description

### DISCUSSION THREAD

**Discussion by daniellrr** [renovatebot/renovate#31446] [Request Help]
### What would you like help with?

I would like help with configuration

### How are you running Renovate?

None

### Please tell us more about your question or problem

Hello,

Im trying to figure out if its possible to append text to the description of an onboarding PR.

What i essentially want is to add a link to our internal documentation page.

As i understand it im able to edit the Onboarding PR:s description but that that would replace the default description, or is there a way to append to the default description? 

Im running Renovate in a scheduled Gitlab CI/CD pipeline with the RENOVATE_ONBOARDING_CONFIG variable configured.

### Logs (if relevant)

```yaml
RENOVATE_ONBOARDING_CONFIG: '{"$$schema": "https://docs.renovatebot.com/renovate-schema.json", "extends": ["config:recommended"], "packageRules": [ { "matchPackagePatterns": ["registry.gitlab.com"], "enabled": false } ] }'

```

**Comment by RahulGautamSingh** [comment]
https://docs.renovatebot.com/configuration-options/#prfooter
